libraries/quantity-picker-view/src/main/res/drawable/qpv_shape_default_background.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <shape xmlns:android="http://schemas.android.com/apk/res/android"
     android:shape="rectangle">
-    <corners android:radius="4dp" />
```

Comment:
Can you revery all changes in this file?

Reply:
I will revert corners and padding since they're kind of arbitrary values but the one with solid is important to render elevation.